src/components/select/filterable-select/filterable-select.stories.tsx

```diff
@@ -384,7 +380,7 @@ export const WithActionButton: Story = () => {
         onCancel={() => setIsOpen(false)}
         title="Dialog component triggered on action"
       >
-        <Button onClick={addNew}>Add new</Button>
+        <Button onClick={addNew}>Add a New Element</Button>
```

Comment:
If we pass `listActionButton` as true, it will render the internal list action button from `list-action-button.component.tsx` which does have an in-built translation:  {l.select.actionButtonText()}.

Otherwise, if it is passed as a node, consumers then have the ability to render whatever they like, putting the onus on them to provide correct translations etc.